LobbyServer2/LobbyServer/LobbyServerProtocol.cs

```diff
@@ -522,6 +874,80 @@ public void HandleClientErrorSummary(ClientErrorSummary request)
 
         public void HandlePreviousGameInfoRequest(PreviousGameInfoRequest request)
         {
+            // This gets called, when they leave a Custom game creation, not sure if theres anything else that gets called to put this in
+            BridgeServerProtocol customServer = ServerManager.GetCustomServerWithPlayer(AccountId);
+            if (customServer != null)
+            {
+                if (customServer.GameInfo.GameServerProcessCode.StartsWith("CustomGame-"))
+                {
+                    // Remove player from Custom game, if no player left delete the CustomGame itself
```

Comment:
Is there nothing else that client sends when leaving a custom game?

Reply:
No don't think so , i could not find anything other then where it is now when it checks for reconnect, the back button does not send any command whatsoever